### DISCUSSION TITLE
Is there a way to view new beta docs in white mode?

### DISCUSSION THREAD

**Discussion by shinebayar-g** [vercel/next.js#43194] [Help]
https://beta.nextjs.org/docs is soo dark.

**Comment by ciruz** [comment]
You can manipulate the CSS, use something like ``Stylish``, or if you use Chrome, you can use an extension.

Something like ``Invert!`` ( https://chrome.google.com/webstore/detail/invert/heidajcpkemganilmhockdniaknoojih ) or the official extension from Google ``High Contrast`` ( https://chrome.google.com/webstore/detail/high-contrast/djcfdncoelnlbldjfhinnjlhdjlikmph ) but the result is not so good with this one, Invert! works better for the docs.
